<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-DH4500R" ignored="0" name="Command line test" tests="61" failures="40" timestamp="03 Jan 2018 21:14:58 GMT" time="0.194" errors="0">
  <testcase name="addItemTest" time="0.02" classname="testng.ItemsTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: nullat testng.ItemsTest.addItemTest(ItemsTest.java:277)... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- addItemTest -->
  <testcase name="addItemTest" time="0.0" classname="testng.ItemsTest"/>
  <testcase name="addItemTest" time="0.0" classname="testng.ItemsTest"/>
  <testcase name="getAllActiveItemsTest" time="0.008" classname="testng.ItemsTest">
    <failure type="java.lang.AssertionError" message="
Expected: every item is hasProperty(&amp;quot;itemIsActive&amp;quot;, is &amp;quot;1&amp;quot;)
     but: was null">
      <![CDATA[java.lang.AssertionError: Expected: every item is hasProperty("itemIsActive", is "1")     but: was nullat org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.ItemsTest.getAllActiveItemsTest(ItemsTest.java:61)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getAllActiveItemsTest -->
  <testcase name="getAllItemsTest" time="0.0" classname="testng.ItemsTest">
    <failure type="java.lang.AssertionError" message="
Expected: every item is an instance of del.res.models.Item
     but: was null">
      <![CDATA[java.lang.AssertionError: Expected: every item is an instance of del.res.models.Item     but: was nullat org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.ItemsTest.getAllItemsTest(ItemsTest.java:166)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getAllItemsTest -->
  <testcase name="getCartItemsTest" time="0.015" classname="testng.ItemsTest">
    <failure type="java.lang.AssertionError" message="
Expected: every item is hasProperty(&amp;quot;itemName&amp;quot;, one of {&amp;quot;Plain Pie&amp;quot;, &amp;quot;Pepperoni Pie&amp;quot;})
     but: was null">
      <![CDATA[java.lang.AssertionError: Expected: every item is hasProperty("itemName", one of {"Plain Pie", "Pepperoni Pie"})     but: was nullat org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.ItemsTest.getCartItemsTest(ItemsTest.java:112)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getCartItemsTest -->
  <testcase name="getCartItemsTest" time="0.007" classname="testng.ItemsTest"/>
  <testcase name="getItemsCountTest" time="0.002" classname="testng.ItemsTest">
    <failure type="java.lang.AssertionError" message="
Expected: a value greater than &amp;lt;0&amp;gt;
     but: &amp;lt;0&amp;gt; was equal to &amp;lt;0&amp;gt;">
      <![CDATA[java.lang.AssertionError: Expected: a value greater than <0>     but: <0> was equal to <0>at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.ItemsTest.getItemsCountTest(ItemsTest.java:72)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getItemsCountTest -->
  <testcase name="getTotalsTest" time="0.002" classname="testng.ItemsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.bo.ItemsBO.getTotals(ItemsBO.java:26)at testng.ItemsTest.getTotalsTest(ItemsTest.java:151)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getTotalsTest -->
  <testcase name="getTotalsTest" time="0.003" classname="testng.ItemsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.bo.ItemsBO.getTotals(ItemsBO.java:26)at testng.ItemsTest.getTotalsTest(ItemsTest.java:151)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getTotalsTest -->
  <testcase name="getTotalsTest" time="0.003" classname="testng.ItemsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.bo.ItemsBO.getTotals(ItemsBO.java:26)at testng.ItemsTest.getTotalsTest(ItemsTest.java:151)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getTotalsTest -->
  <testcase name="updateItemTest" time="0.0" classname="testng.ItemsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.ItemsDAO.getItemByID(ItemsDAO.java:238)at del.res.bo.ItemsBO.getItemByID(ItemsBO.java:70)at testng.ItemsTest.updateItemTest(ItemsTest.java:202)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- updateItemTest -->
  <testcase name="updateItemTest" time="0.001" classname="testng.ItemsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.ItemsDAO.getItemByID(ItemsDAO.java:238)at del.res.bo.ItemsBO.getItemByID(ItemsBO.java:70)at testng.ItemsTest.updateItemTest(ItemsTest.java:202)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- updateItemTest -->
  <testcase name="updateItemTest" time="0.001" classname="testng.ItemsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.ItemsDAO.getItemByID(ItemsDAO.java:238)at del.res.bo.ItemsBO.getItemByID(ItemsBO.java:70)at testng.ItemsTest.updateItemTest(ItemsTest.java:202)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- updateItemTest -->
  <testcase name="addItemToOrderTest" time="0.0" classname="testng.OrdersTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: nullat testng.OrdersTest.addItemToOrderTest(OrdersTest.java:119)... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- addItemToOrderTest -->
  <testcase name="addItemToOrderTest" time="0.0" classname="testng.OrdersTest"/>
  <testcase name="addItemToOrderTest" time="0.0" classname="testng.OrdersTest"/>
  <testcase name="addItemToOrderTest" time="0.0" classname="testng.OrdersTest"/>
  <testcase name="createOrderTest" time="0.0" classname="testng.OrdersTest">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;true&amp;gt;
     but: was &amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError: Expected: <true>     but: was <false>at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.OrdersTest.createOrderTest(OrdersTest.java:80)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createOrderTest -->
  <testcase name="getAllOrdersTest" time="0.0" classname="testng.OrdersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.OrdersDAO.getAllOrders(OrdersDAO.java:138)at del.res.bo.OrdersBO.getAllOrders(OrdersBO.java:31)at testng.OrdersTest.getAllOrdersTest(OrdersTest.java:171)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getAllOrdersTest -->
  <testcase name="getOrdersByUserTest" time="0.0" classname="testng.OrdersTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: nullat testng.OrdersTest.getOrdersByUserTest(OrdersTest.java:158)... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- getOrdersByUserTest -->
  <testcase name="getOrdersByUserTest" time="0.0" classname="testng.OrdersTest"/>
  <testcase name="getOrdersByUserTest" time="0.016" classname="testng.OrdersTest"/>
  <testcase name="getReceiptItemsTest" time="0.0" classname="testng.ReceiptsTest"/>
  <testcase name="getReceiptItemsTest" time="0.0" classname="testng.ReceiptsTest">
    <failure type="java.lang.AssertionError" message="
Expected: an empty collection
     but: was null">
      <![CDATA[java.lang.AssertionError: Expected: an empty collection     but: was nullat org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.ReceiptsTest.getReceiptItemsTest(ReceiptsTest.java:128)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getReceiptItemsTest -->
  <testcase name="getReceiptSummaryTest" time="0.016" classname="testng.ReceiptsTest">
    <failure type="java.lang.AssertionError" message="
Expected: same property values as ReceiptSummary [itemCount: &amp;lt;2&amp;gt;, orderID: null, orderTax: &amp;quot;$2.58&amp;quot;, orderTotal: &amp;quot;$32.08&amp;quot;, storeAddress: &amp;quot;Bronx, NY&amp;quot;, storeDescription: &amp;quot;Frankies Pizza&amp;quot;]
     but: was null">
      <![CDATA[java.lang.AssertionError: Expected: same property values as ReceiptSummary [itemCount: <2>, orderID: null, orderTax: "$2.58", orderTotal: "$32.08", storeAddress: "Bronx, NY", storeDescription: "Frankies Pizza"]     but: was nullat org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.ReceiptsTest.getReceiptSummaryTest(ReceiptsTest.java:72)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getReceiptSummaryTest -->
  <testcase name="validateUserTest" time="0.0" classname="testng.ReceiptsTest">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;lt;true&amp;gt;
     but: was &amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError: Expected: <true>     but: was <false>at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.ReceiptsTest.validateUserTest(ReceiptsTest.java:100)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- validateUserTest -->
  <testcase name="getItemInfoTest" time="0.0" classname="testng.ReviewsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.ReviewsDAO.getItemInfo(ReviewsDAO.java:46)at del.res.bo.ReviewsBO.getItemInfo(ReviewsBO.java:17)at testng.ReviewsTest.getItemInfoTest(ReviewsTest.java:104)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getItemInfoTest -->
  <testcase name="getPastReviewsTest" time="0.0" classname="testng.ReviewsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.ReviewsDAO.getPastReviews(ReviewsDAO.java:73)at del.res.bo.ReviewsBO.getPastReviews(ReviewsBO.java:22)at testng.ReviewsTest.getPastReviewsTest(ReviewsTest.java:130)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getPastReviewsTest -->
  <testcase name="getReviewsByUserTest" time="0.0" classname="testng.ReviewsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.ReviewsDAO.getReviewsByUser(ReviewsDAO.java:99)at del.res.bo.ReviewsBO.getReviewsByUser(ReviewsBO.java:27)at testng.ReviewsTest.getReviewsByUserTest(ReviewsTest.java:163)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getReviewsByUserTest -->
  <testcase name="submitReviewTest" time="0.0" classname="testng.ReviewsTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.ReviewsDAO.submitReview(ReviewsDAO.java:20)at del.res.bo.ReviewsBO.submitReview(ReviewsBO.java:12)at testng.ReviewsTest.submitReviewTest(ReviewsTest.java:65)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- submitReviewTest -->
  <testcase name="createStoreTest" time="0.001" classname="testng.StoresTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.StoresDAO.createStore(StoresDAO.java:96)at del.res.bo.StoresBO.createStore(StoresBO.java:22)at testng.StoresTest.createStoreTest(StoresTest.java:124)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createStoreTest -->
  <testcase name="getAllStoresTest" time="0.002" classname="testng.StoresTest">
    <failure type="java.lang.AssertionError" message="
Expected: every item is an instance of del.res.models.Store
     but: was null">
      <![CDATA[java.lang.AssertionError: Expected: every item is an instance of del.res.models.Store     but: was nullat org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.StoresTest.getAllStoresTest(StoresTest.java:49)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- getAllStoresTest -->
  <testcase name="updateStoreTest" time="0.0" classname="testng.StoresTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.StoresDAO.getStoreByID(StoresDAO.java:125)at del.res.bo.StoresBO.getStoreByID(StoresBO.java:33)at testng.StoresTest.updateStoreTest(StoresTest.java:86)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- updateStoreTest -->
  <testcase name="createUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at del.res.bo.UsersBO.createUser(UsersBO.java:20)at testng.UsersTest.createUserTest(UsersTest.java:94)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase name="createUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at del.res.bo.UsersBO.createUser(UsersBO.java:20)at testng.UsersTest.createUserTest(UsersTest.java:94)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase name="createUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at del.res.bo.UsersBO.createUser(UsersBO.java:20)at testng.UsersTest.createUserTest(UsersTest.java:94)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase name="createUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at del.res.bo.UsersBO.createUser(UsersBO.java:20)at testng.UsersTest.createUserTest(UsersTest.java:94)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase name="createUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at del.res.bo.UsersBO.createUser(UsersBO.java:20)at testng.UsersTest.createUserTest(UsersTest.java:94)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase name="createUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at del.res.bo.UsersBO.createUser(UsersBO.java:20)at testng.UsersTest.createUserTest(UsersTest.java:94)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase name="createUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at del.res.bo.UsersBO.createUser(UsersBO.java:20)at testng.UsersTest.createUserTest(UsersTest.java:94)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase name="createUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at del.res.bo.UsersBO.createUser(UsersBO.java:20)at testng.UsersTest.createUserTest(UsersTest.java:94)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase name="createUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at del.res.bo.UsersBO.createUser(UsersBO.java:20)at testng.UsersTest.createUserTest(UsersTest.java:94)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- createUserTest -->
  <testcase name="getDetailsTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: nullat testng.UsersTest.getDetailsTest(UsersTest.java:154)... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- getDetailsTest -->
  <testcase name="getDetailsTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="getDetailsTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="loginQueryTest" time="0.015" classname="testng.UsersTest">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;quot;3&amp;quot;
     but: was null">
      <![CDATA[java.lang.AssertionError: Expected: "3"     but: was nullat org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)at testng.UsersTest.loginQueryTest(UsersTest.java:123)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- loginQueryTest -->
  <testcase name="loginQueryTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="loginQueryTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="loginQueryTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="loginQueryTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="registerQueryTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at testng.UsersTest.registerQueryTest(UsersTest.java:62)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- registerQueryTest -->
  <testcase name="registerQueryTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at testng.UsersTest.registerQueryTest(UsersTest.java:62)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- registerQueryTest -->
  <testcase name="registerQueryTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat del.res.dao.UsersDAO.registerQuery(UsersDAO.java:21)at del.res.bo.UsersBO.registerQuery(UsersBO.java:16)at testng.UsersTest.registerQueryTest(UsersTest.java:62)... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- registerQueryTest -->
  <testcase name="updateUserTest" time="0.0" classname="testng.UsersTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: nullat testng.UsersTest.updateUserTest(UsersTest.java:207)... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- updateUserTest -->
  <testcase name="updateUserTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="updateUserTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="updateUserTest" time="0.016" classname="testng.UsersTest"/>
  <testcase name="updateUserTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="updateUserTest" time="0.0" classname="testng.UsersTest"/>
  <testcase name="updateUserTest" time="0.0" classname="testng.UsersTest"/>
</testsuite> <!-- Command line test -->
